packages/block-library/src/navigation/edit/use-create-navigation-menu.js

```diff
@@ -60,7 +63,7 @@ export default function useCreateNavigationMenu( clientId ) {
 			const record = {
 				title,
 				content: serialize( blocks ),
-				status: 'publish',
+				status: postStatus,
```

Comment:
I'm sure there were [a few instances in the block where we only treat `publish` posts as valid](https://github.com/WordPress/gutenberg/pull/42982). How does this PR effect this behaviour?

Reply:
The menu selector only shows published posts (this forces menu duplication, as they cannot choose the classic menu created once they leave the page). Maybe I can put here both `publish` and `draft` and check the warnings again.

![Screenshot 2022-08-24 at 16 28 48](https://user-images.githubusercontent.com/37012961/186444858-ef306e72-44ac-451e-823e-bee03f8a3d69.jpg)


https://github.com/WordPress/gutenberg/blob/d2c46bfb01c2e1dc8f15512d747865b88a151108/packages/block-library/src/navigation/use-navigation-menu.js#L74

Reply:
I tried to fix this in the last commit. Let me know if this is a good approach!